factories/caseStatus.ts

```diff
@@ -12,9 +17,20 @@ export const mockedCaseStatusFactory = Factory.define<CaseStatus>(
   ({ sequence }) => ({
     id: sequence,
     type: 'foo',
-    subType: 'bar',
-    StartDate: '2021-1-1',
-    EndDate: '2021-1-1',
-    Notes: 'a lot to note',
+    fields: [mockedStatusField.build()],
+    startDate: '2021-1-1',
+    endDate: '2021-1-1',
```

Comment:
Are we going to be returning ISO formatted date strings from our API (I hope so).

If so we should probably make sure our mocked strings are in ISO-8601 format.

Reply:
Changed them to ISO